docs/guides/Quick Start/03-orders.md

```diff
@@ -0,0 +1,182 @@
+# Order Overview
+
+The order resource enables merchants to process orders they receive from customers.
+
+You can use the orders resource to do the following:
+
+* Retrieve orders and their line-items
+* Create dummy orders 
+* Update order statuses
+* Manage returns
+
+## Retrieve Orders
+
+A list of all customer orders is obtained using the route:
+
+``` markdown
+POST /api/search/order
+```
```

Comment:
```suggestion
```json http
{
  "method": "post",
  "url": "/api/search/order",
  "headers": {
    "Content-Type": "application/json",
    "Accept": "application/json",
    "Authorization": "Bearer Your_API_Key"
  }
}
```
```

Reply:
You can also mention, that sorting, aggregating and narrowing down order results is possible using a search criteria and referencing the corresponding section about [search queries](https://shopware.stoplight.io/docs/store-api/cf710bf73d0cd-search-queries).